crates/modelardb_manager/src/remote.rs

```diff
@@ -60,7 +63,17 @@ pub fn start_apache_arrow_flight_server(
 /// published under Apache2.
 ///
 /// [Apache Arrow Flight examples]: https://github.com/apache/arrow-rs/blob/master/arrow-flight/examples
-struct FlightServiceHandler {}
+#[allow(dead_code)]
```

Comment:
Maybe add a comment why `#[allow(dead_code)]` is needed and when it can be removed.

Reply:
It has already been removed in the next branch so I have just kept it as it is.